easy-ui-react/src/Notification/Notification.mdx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Canvas, Meta, Controls, ArgTypes } from "@storybook/blocks";
+import { NotificationProvider } from "./Notification";
+import * as NotificationStories from "./Notification.stories";
+
+<Meta of={NotificationStories} />
+
+# Notification
+
+## Notification
+
+Notifications of type `alert` and `toast` can be displayed via Easy UI's `useNotification` hook. The hook returns a notification
+object on which methods can be invoked to display the appropriate notification. Use the `Show code` tab below for the associated code sample.
+
+<Canvas of={NotificationStories.Notifications} />
+
+## Custom placement
+
+Notifications can have custom placement on a per app basis. Use the `notificationPlacementOffset` prop which is available on the `EasyUIProvider` component.
+
+<Canvas of={NotificationStories.CustomPlacement} />
+
+## Properties
```

Comment:
i know we've been using `ArgsTable` here, but with the base `Notification` component, the output included the `state` and `toast` prop, which i think has the potential to become a little confusing since those aren't relevant to consumers, but open to ideas/thoughts